shared/middleware/files_api.rb

```diff
@@ -719,6 +719,17 @@ def files_put_file(encrypted_channel_id, filename, body)
   end
 
   #
+  # GET /v3/files-public/<channel-id>/.metadata/<filename>?version=<version-id>
+  #
+  # Read a metadata file, caching the result for 1 hour.
+  #
+  get %r{/v3/files-public/([^/]+)/.metadata/([^/]+)$} do |encrypted_channel_id, filename|
+    file = get_file('files', encrypted_channel_id, "#{METADATA_PATH}/#{filename}")
+    cache_for 1 * 60 * 60 # 1 hour
```

Comment:
Can we write `cache_for 1.hour` here, or do we not have access to [ActiveSupport::CoreExtensions::Numeric::Time](http://api.rubyonrails.org/v2.3/classes/ActiveSupport/CoreExtensions/Numeric/Time.html#M000596) in our shared middleware?